upstream-community-operators/snyk-operator/1.30.6/snyk-operator.v1.30.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,259 @@
+      mediatype: image/svg+xml
+  provider:
+    name: Snyk Ltd.
+  maintainers:
+    - name: Snyk Ltd.
+      email: support@snyk.io
+  links:
+    - name: Website
+      url: https://snyk.io
+    - name: Documentation
+      url: https://support.snyk.io/hc/en-us/articles/360003916138-Kubernetes-integration-overview
+  version: 1.30.6
+  keywords:
+    - 'snyk'
+    - 'controller'
+    - 'monitor'
+    - 'operator'
+    - 'security'
+    - 'cluster'
+    - 'container'
+    - 'image'
+    - 'security'
+    - 'vulnerabilities'
+    - 'advisories'
+    - 'audit'
+    - 'scan'
+    - 'scanning'
+  replaces: snyk-operator.v1.30.0
```

Comment:
The erroneous reference to the non-existent previous version of the Operator is still there.

Reply:
Hi @dmesser, I've just updated that! Thanks for pointing out the problem.